src/components/footer/footer.scss

```diff
@@ -0,0 +1,34 @@
+@import '~theme/base/variables';
+@import '~theme/base/mediaqueries';
+@import '~theme/base/utils';
+
+.footerMain {
+  width: 100%;
+}
+
+.footerInner {
+  padding: 1em 0;
```

Comment:
Can we be consistent and use `px` for margin & padding, please? :)

Reply:
Oups, I think I saw em in someone else's paddings and got it from there. I'll fix those ones too then ;-). (The proofreader that secretly lives in me is excited)

Reply:
I've been trolling everyone else too, don't worry!